src/main-process/atom-application.coffee

```diff
@@ -322,6 +324,8 @@ class AtomApplication
 
     @disposable.add ipcHelpers.on ipcMain, 'did-cancel-window-unload', =>
       @quitting = false
+      for window in @windows
+        window.didCancelWindowUnload()
```

Comment:
Oh, and we should probably call it something else to avoid shadowing `window` global? Better safe than sorry 🙈